.github/workflows/create-branch.yml

```diff
@@ -0,0 +1,27 @@
+name: Auto Create Branch
+
+on:
+  workflow_call:
+
+permissions:
+  contents: write
+
+jobs:
+  create-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Create Branch name
+        id: branch_name
+        run: |
+          issue_number="${{ github.event.issue.number }}"
+          issue_title=$(echo "${{ github.event.issue.title }}" | tr '[:upper:]' '[:lower:]' | sed 's/ /-/g')
+
+          echo "BRANCH_NAME=i${issue_number}_${issue_title}" >> $GITHUB_ENV
+      - name: Create Issue Branch
+        if: "!contains($(git ls-remote --heads origin ${env.BRANCH_NAME}), ${{ env.BRANCH_NAME }})"
+        uses: peterjgrainger/action-create-branch@v2.0.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          branch: "${{ env.BRANCH_NAME }}"
```

Comment:
This seems to be one simple api call, so maybe its better to implement it by our own.